doc-content/enterprise-only/integration/entando-con.adoc

```diff
@@ -4,6 +4,8 @@
 
 Digital Experience Platform (DXP) is a low-code, component-based platform for building modern web, mobile, and IoT applications. AppBuilder is an application that provides features to help you configure and customize, with or without programming skills, any kind of modern web application built with AppBuilder. AppBuilder is a web-based administration console that you can use to build and configure your web application, manage your page data and activities, and integrate with other application tools, such as {PRODUCT}.
 
+{PRODUCT} includes a rich, open-source user experience platform for building modern, process-driven applications in the cloud. AppBuilder uses the cloud native infrastructure provided by Openshift leveraging services like ingress, sidecar and pipelines. The user experience platform streamlines the development and allows business users to create compelling experiences for customers across a wide range of devices and channels. Business experts can quickly compose the applications by dragging and dropping the reusable user interface components to pages. Developers can freely work in the frontend frameworks of their choice. AppBuilder provides a library of out-of-the-box and Micro-Frontend widgets which enables the task lists, forms and process graphs.
```

Comment:
OpenShift or {OPENSHIFT}
like - such as

enables business users
micro-frontend
forms, and process graphs

ingress, sidecar and pipelines - are any of these proper nouns?

Reply:
ingress is proper noun.
It will change it to: Ingress

sidecar is the name of the container